sec3/neovim.md

```diff
@@ -0,0 +1,207 @@
+## Neovim: a Deeper Dive into vi/Vim
+
+All the steps included in the [vi](vi.md) will work in Neovim. We will take a deeper dive in using a modern implementation of vi in Neovim. We will look at
```

Comment:
So far everything in this file is true about vim as well as neovim. I think it would be more general to replace instances of neovim with vim.

I think plugins can be different in neovim so that section might need to be more specific once it's developed.